packages/forma-36-website/src/components/MDXPage.js

```diff
@@ -150,57 +110,86 @@ export default function MDXPage({ mdxContent, frontmatter, propsMetadata }) {
     <Box className={styles.pageContent}>
       <PropsProvider metadata={propsMetadata}>
         <MDXProvider components={markToComponentMap}>
-          <header className={styles.header}>
-            {frontmatter?.title && (
-              <DisplayText marginBottom="spacingL">
-                {frontmatter.title}
-              </DisplayText>
-            )}
+          {!frontmatter.v4Doc && (
+            <header className={styles.header}>
+              {frontmatter?.title && (
+                <DisplayText marginBottom="spacingL">
+                  {frontmatter.title}
+                </DisplayText>
+              )}
 
-            <div className={styles.subheaderRow}>
-              <div className={styles.buttonList}>
-                {frontmatter?.storybook && (
-                  <a
-                    className={styles.imageLink}
-                    href={frontmatter.storybook}
-                    title={`View ${frontmatter?.title} in Storybook`}
-                  >
-                    <img src={storybookIcon} alt="" />
-                    <span>Storybook</span>
-                  </a>
-                )}
-                {frontmatter?.github && (
-                  <a
-                    className={styles.imageLink}
-                    href={frontmatter.github}
-                    title={`View ${frontmatter?.title} on GitHub`}
-                  >
-                    <img src={githubIcon} alt="" />
-                    <span>Github</span>
-                  </a>
+              <div className={styles.subheaderRow}>
```

Comment:
@gui-santos I think in that case we don't want this subheader row with storybook, github and status?

Reply:
but it is needed for the pages that are still with the old format
everything from line 113 to line 162 is used when `frontmatter.v4Doc === false`

In this PR, only the page for Accordion has `v4Doc = true`

Reply:
aaa, gotya! oki!